extensions/http/jersey/src/main/java/org/eclipse/dataspaceconnector/extension/jersey/mapper/EdcApiExceptionMapper.java

```diff
@@ -66,8 +67,11 @@ public Response toResponse(Throwable exception) {
 
         if (exception instanceof EdcException && verboseResponse) {
             var edcApiException = (EdcException) exception;
-            var responseBody = List.of(Map.of("message", edcApiException.getMessage()));
-            return Response.status(status).entity(responseBody).build();
+            var apiError = ApiErrorDetail.Builder.newInstance()
+                    .message(edcApiException.getMessage())
+                    .type(edcApiException.getClass().getSimpleName().replace("Exception", ""))
+                    .build();
+            return Response.status(status).entity(List.of(apiError)).build();
```

Comment:
any particular reason this needs to be a `List`?

Reply:
@paullatzelsperger this because the `ConstraintViolationException` contains a list of violations (as a validation could fail for different reasons), so the responses are consistent.

Reply:
ah I see. makes sense.